src/test/groovy/com/jvm_bloggers/core/data_fetching/blogs/json_data/JsonBlogsDataFileValidationSpec.groovy

```diff
@@ -1,59 +1,51 @@
 package com.jvm_bloggers.core.data_fetching.blogs.json_data
 
 import com.google.common.base.CharMatcher
-import com.jvm_bloggers.JvmBloggersApplication
 import groovy.json.JsonSlurper
-import org.springframework.beans.factory.annotation.Value
-import org.springframework.boot.test.context.SpringBootTest
-import org.springframework.core.io.Resource
-import org.springframework.test.context.ActiveProfiles
-import org.springframework.test.context.ContextConfiguration
+import spock.lang.Shared
 import spock.lang.Specification
 import spock.lang.Subject
+import spock.lang.Unroll
 
-@ContextConfiguration
-@SpringBootTest(classes = JvmBloggersApplication)
-@ActiveProfiles("test")
 class JsonBlogsDataFileValidationSpec extends Specification {
 
-    @Value("classpath:blogs/bloggers.json")
-    Resource bloggersJson
+    @Subject
+    @Shared
+    List<BloggerEntry> jsonEntries
 
-    @Value("classpath:blogs/companies.json")
-    Resource companiesJson
+    private static final int TWITTER_USER_TAG_MAXIMUM_LENGTH = 16
```

Comment:
"Your username cannot be longer than 15 characters." from https://help.twitter.com/en/managing-your-account/twitter-username-rules

Reply:
I named it as USER_TAG to distinguish from username. We store them @ + username, so 16. Should I name it better?

Reply:
Maybe `AT_PLUS_TWITTER_HANDLE_MAX_LENGTH`?